### ISSUE TITLE
Get losses fail 

### ISSUE THREAD

**Issue by Miriam2040** [EtienneT/TabularVAE#2]
I am using this code to run AE for anomaly detection. 

I am trying to get the losses to see who are my outliers:
`probs, targs, loss = learn.get_preds(with_loss=True)`

getting this error:
forward() got an unexpected keyword argument 'reduction'

how should I fix it?

**Comment by EtienneT** [comment]
Oh very interesting use case.  I did a new commit to fix that.  Basically RecreatedLoss did not have a reduction parameter.  The loss was returning the mean loss for the whole batch.  Now I implemented a reduction parameter where reduction='none' will return the loss for each item in the batch.

So learn.get_preds(with_loss=True) works now.

**Comment by Miriam2040** [comment]
> Oh very interesting use case. I did a new commit to fix that. Basically RecreatedLoss did not have a reduction parameter. The loss was returning the mean loss for the whole batch. Now I implemented a reduction parameter where reduction='none' will return the loss for each item in the batch.
> 
> So learn.get_preds(with_loss=True) works now.

Thanks!
